authority-portal-backend/authority-portal-api/src/main/java/de/sovity/authorityportal/api/model/ConnectorTypeDto.kt

```diff
@@ -19,5 +19,6 @@ import io.swagger.v3.oas.annotations.media.Schema
 enum class ConnectorTypeDto {
     OWN,
     PROVIDED,
-    CAAS
+    CAAS,
+    CONFIGURING
```

Comment:
I would argue for `CONFIGURING` here because the status is shown to the customer organization which cannot actually configure the connector (hence "configurable" would be wrong).

On the other side, the provider doesn't see the status if they don't click on details, because instead of the status, they get a big "Configure" button.

Customer:
<img width="1454" alt="image" src="https://github.com/user-attachments/assets/eeebe018-747b-492c-bb92-cf7721de67db">

Provider:
<img width="1456" alt="image" src="https://github.com/user-attachments/assets/2511db66-796c-4c70-a46b-380307498771">